woocommerce-services.php

```diff
@@ -361,6 +361,7 @@ public function load_textdomain() {
 		public function init() {
 			$this->load_dependencies();
 			add_action( 'admin_init', array( $this, 'admin_enqueue_scripts' ) );
+			add_action( 'admin_init', array( $this->nux, 'set_up_nux_notices' ) );
```

Comment:
@jeffstieler - I don't think I  understand where you'd like this to go after all. Can you elaborate?